docs/app/Examples/elements/List/Content/ListHeaderExample.js

```diff
@@ -8,6 +8,16 @@ export default class ListHeaderExample extends Component {
         <List.Item header='Chapter 1' description='The chapter in which we meet the characters' />
         <List.Item header='Chapter 2' description='The chapter in which the bad guy is introduced' />
         <List.Item header='Chapter 3' description='Spoiler alert: The chapter in which the good guy wins!' />
+        <List.Item>
+          <List.ItemContent>
```

Comment:
When there is no image/icon do we want to require ItemContent? The current ListItem file suggests no.
ListItem.js: if (media) content = <div className='content'>{content}</div>